src/main/java/br/com/annuum/capsicum/api/mapper/FederatedUnityMapper.java

```diff
@@ -0,0 +1,24 @@
+package br.com.annuum.capsicum.api.mapper;
+
+import br.com.annuum.capsicum.api.domain.dto.FederatedUnityDto;
+import br.com.annuum.capsicum.api.domain.FederatedUnity;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class FederatedUnityMapper {
```

Comment:
Implementando o `br.com.annuum.capsicum.api.mapper.Mapper<IN, OUT>` não precisaria do map da lista